src/crypto/crypto_context.h

```diff
@@ -43,13 +44,27 @@ class SecureContext final : public BaseObject {
 
   const SSLCtxPointer& ctx() const { return ctx_; }
 
+  template <typename Func = void(SSLCtxPointer&)>
+  inline void Initialize(Func fn) {
+    fn(ctx_);
+  }
+
+  inline X509Pointer& issuer() { return issuer_; }
+  inline X509Pointer& cert() { return cert_; }
+
   SSLPointer CreateSSL();
 
   void SetGetSessionCallback(GetSessionCb cb);
   void SetKeylogCallback(KeylogCb cb);
   void SetNewSessionCallback(NewSessionCb cb);
   void SetSelectSNIContextCallback(SelectSNIContextCb cb);
 
+  bool AddCert(BIOPointer&& bio);
+  void SetCACert(const BIOPointer& bio);
+  void SetRootCerts();
+  bool SetCRL(const BIOPointer& bio);
+  bool UseKey(std::shared_ptr<KeyObjectData> key);
```

Comment:
Consider adding MUST_USE_RESULT to the methods returning bool.

Reply:
Handled in #45912